Firestore/core/src/model/object_value.h

```diff
@@ -37,26 +37,22 @@ namespace firestore {
 namespace model {
 
 /** A structured object value stored in Firestore. */
-// TODO(mutabledocuments): Rename to ObjectValue once other ObjectValue class
-// is removed
-class MutableObjectValue {
+class ObjectValue {
  public:
-  MutableObjectValue();
+  ObjectValue();
 
   /** Creates a new MutableObjectValue and takes ownership of `value`. */
-  explicit MutableObjectValue(const google_firestore_v1_Value& value)
-      : value_(value) {
-    HARD_ASSERT(
-        value.which_value_type == google_firestore_v1_Value_map_value_tag,
-        "ObjectValues should be backed by a MapValue");
-  }
+  explicit ObjectValue(const google_firestore_v1_Value& value);
 
-  MutableObjectValue(MutableObjectValue&& other) noexcept = default;
-  MutableObjectValue& operator=(MutableObjectValue&& other) = default;
+  ObjectValue(ObjectValue&& other) noexcept = default;
+  ObjectValue& operator=(ObjectValue&& other) = default;
```

Comment:
Nit: `operator=` can be `noexcept` as well.

Reply:
Done